api/hypershift/v1beta1/nodepool_types.go

```diff
@@ -833,6 +833,14 @@ type AWSNodePoolPlatform struct {
 	// +optional
 	ResourceTags []AWSResourceTag `json:"resourceTags,omitempty"`
 
+	// Placement specifies the placement options for the EC2 instances.
```

Comment:
Joel has mentioned to me in the past the godoc should be lowercase for the field here in the comment.

cc: @JoelSpeed in case you want to check out this new API field.

Reply:
can you elaborate more? What exactly should be lowercase? The `Placement` word?

Reply:
Yeah `placement`

Reply:
I'd rather we stay consistent across all fields, which are now all using upperCase. 
Maybe, we could have a ticket to change all to lowerCase.

Reply:
The godoc should start with the `json` tag version, ie, the serialised version, since that's what the end user would normally interact with, and these docs are for end users to understand the field.

> I'd rather we stay consistent across all fields, which are now all using upperCase.

This perpetuates the incorrect version and increases what there is to fix later. If you start fixing now, you have fewer to fix later.

Reply:
fair enough, updated.